scala/ppml/src/main/java/com/intel/analytics/bigdl/ppml/common/FLDataType.java

```diff
@@ -19,6 +19,6 @@
 public enum FLDataType {
     TENSOR_MAP,
     TREE_SPLIT,
-    TREE_LEAF,
+    TREE_LEAVES,
```

Comment:
Why TREE_LEAVES instead of TREE_LEAF?

Reply:
This actually store an array of leaves. However this is just a naming and we'd better fix all the names the same.

Reply:
treeLeavesStorage sounds very awkward

Reply:
How about change the type to `TREE_LEAF` and `getTreeLeafStorage()` and remain the proto `TreeLeaves'

Reply:
I think we should keep them consistent - either LEAF or LEAVES for all

Reply:
So change to LEAF for all?